src/main/java/uk/gov/hmcts/ccd/data/user/DefaultUserRepository.java

```diff
@@ -179,7 +180,9 @@ public String getUserId() {
     public List<String> getUserRolesJurisdictions() {
         String[] roles = this.getUserDetails().getRoles();
 
-        return Arrays.stream(roles).map(role ->  role.split("-"))
+        return Arrays.stream(roles)
```

Comment:
I was thinking whether we can take this opportunity to also make the method a bit more precise. 
With the current implementation any role of the form part1-part2-....partN will be considered a "jurisidictional" role for a jurisdiction "part2".
In reality this (not written) rule applies only for caseworkers (as far as I know).
So was wondering if it would be better applying this method only to caseworkers roles (maybe using a regex?), and rename it to `getCaseworkerUserRolesJurisdictions`?